javascript/extractor/tests/esnext/output/trap/yield-import.js.trap

```diff
@@ -9,169 +9,35 @@ hasLocation(#10000,#10002)
 #20000=@"global_scope"
 scopes(#20000,0)
 #20001=@"script;{#10000},1,1"
-#20002=*
-lines(#20002,#20001,"function* f() {","
+toplevels(#20001,0)
+#20002=@"loc,{#10000},1,1,1,1"
+locations_default(#20002,#10000,1,1,1,1)
+hasLocation(#20001,#20002)
+#20003=*
+jsParseErrors(#20003,#20001,"Error: Unexpected token","  yield import(""foo"")
```

Comment:
This doesn't look right; that syntax error shouldn't be there on the rc.

Reply:
*Sigh*. That was a case of Eclipse needing a refresh